src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = (
+  { person: { name, age, sex, isMarried, partnerName } },
+) => {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age ? (<p className="Person__age">{`I am ${age}`}</p>) : ''}
```

Comment:
better way is using && 
f.e. age && <p className...>

if age === false, tag <p> even shouldn't be render and add to DOM 


i had some problems with tests for it, so be careful